src/java/org/apache/cassandra/auth/IRoleManager.java

```diff
@@ -152,6 +152,15 @@ void revokeRole(AuthenticatedUser performer, RoleResource role, RoleResource rev
                                        .collect(Collectors.toSet());
      }
 
+    /**
+     * Return the set of roles that are member of the given role.
+     * @return any non-{@code null} value means the found reverse-memberships.
+     *         {@code null} means that the functionality is not supported.
+     * @throws RequestValidationException
+     * @throws RequestExecutionException
+     */
+    Set<RoleResource> getRoleMemberOf(RoleResource role);
```

Comment:
I never understood this function name. We probably shouldn't change it to avoid conflicts with upstream, but what's wrong with `getMembersOf` ?

Reply:
I agree about the confusing name and like `getMembersOf`. This being ported, so there shouldn't be any conflict with upstream OSS, right?